metaflow/plugins/kfp/tests/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM hsezhiyan/metaflow-integration-testing:1.0
```

Comment:
Check with Yun on best container image approach.  
See if we need a JIRA ticket for our OSS Docker files.

Reply:
How do we publish the `metaflow-kfp` Docker file to Dockerhub?

Reply:
We use the `docker login` and then `docker push` commands. This is inherently secure because the password in `docker login` is available in plain view, and I'll investigate ways to keep it hidden.

Reply:
No longer using this image - sticking with `metaflow-zillow`.

Reply:
Final update: we've switched over to `analytics-docker.artifactory.zgtools.net/artificial-intelligence/ai-platform/metaflow-zillow:2.1` to move over from Dockerhub because of limit rates.